features/repo/azure_devops.feature

```diff
@@ -0,0 +1,26 @@
+@skipWindows
+Feature: Azure DevOps
+
+  Scenario Outline:
+    Given the origin is "<ORIGIN>"
+    And tool "open" is installed
+    When I run "git-town repo"
+    Then "open" launches a new pull request with this url in my browser:
+      """
+      https://dev.azure.com/organization/repository
```

Comment:
This is supposed to be the web page for the repo on Azure DevOps. Similar to https://github.com/git-town/git-town for the Git Town codebase.

Reply:
Sorry it's a bit more convoluted than this.

- Here's a good reference [lazygit](https://github.com/jesseduffield/lazygit/blob/b17c38befdd8ea6d4f71353391deb33fd6cc7176/pkg/commands/hosting_service/definitions.go#L45)

1. Source from and to can be pre-populated
2. SSH url is different from the https based url
3. Many still use the visualstudio.com domain even though the dev.azure.com is the more modern and probably the best one to use as it's future proof, if the url isn't be parsed via the underlying provider. If it is it will need to recognize both azure devops + visualstudio.com urls even if the end url is hard coded to the dev.azure.com.